### PR TITLE
Fix typo in name of test for generating coverage

### PR DESCRIPTION
**Pull request by Koncpa**
Signed-off-by: Patrik Koncity <pkoncity@redhat.com>

### CODE REVIEW

**Comment by Koncpa** [comment]
Related to https://github.com/RedHat-SP-Security/keylime-tests/pull/343.

**Review by aplanas — APPROVED** [review]

File: packit-ci.fmf [hunk 1]
```diff
@@ -42,20 +42,22 @@
...
        - /functional/db-mariadb-sanity-on-localhost
        - /functional/db-mysql-sanity-on-localhost
+       - /functional/durable-attestion-sanity-on-localhost
        - /functional/ek-cert-use-ek_check_script
        - /functional/ek-cert-use-ek_handle-custom-ca_certs
        - /functional/install-rpm-with-ima-signature
        - /functional/keylime-non-default-ports
        - /functional/keylime_tenant-commands-on-localhost
        - /functional/keylime_tenant-ima-signature-sanity
+       - /functional/measured-boot-policy-sanity
```
Comment (kkaarreell) [inline packit-ci.fmf:52]: This one should not be here. I mean, it won't be executed anyway since it requires real TPM device on a system.

**Comment by aplanas** [comment]
All green here. That should be ready to merge, or it is missing something?

**Comment by ansasaki** [comment]
> All green here. That should be ready to merge, or it is missing something?

It is all right, merging now

**Pull request merged**